SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceEventBuilder.cpp

```diff
@@ -100,25 +100,27 @@ FSpatialTraceEvent FSpatialTraceEventBuilder::GetEvent() &&
 	return MoveTemp(SpatialTraceEvent);
 }
 
-FSpatialTraceEvent FSpatialTraceEventBuilder::CreateProcessRPC(const UObject* Object, UFunction* Function)
+FSpatialTraceEvent FSpatialTraceEventBuilder::CreateProcessRPC(const UObject* Object, UFunction* Function,
+															   const EventTraceUniqueId& LinearTraceId)
 {
 	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "process_rpc")
 		.AddObject(TEXT("Object"), Object)
 		.AddFunction(TEXT("Function"), Function)
+		.AddKeyValue(TEXT("LinearTraceId"), LinearTraceId.ToString())
 		.GetEvent();
 }
 
-FSpatialTraceEvent FSpatialTraceEventBuilder::CreateSendRPC(const UObject* Object, UFunction* Function)
+FSpatialTraceEvent FSpatialTraceEventBuilder::CreatePushRPC(const UObject* Object, UFunction* Function)
 {
-	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "send_rpc")
+	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "push_rpc")
```

Comment:
Btw what's the impact to our data / current NFRs by having `push_rpc` now in our traces?

Reply:
So no impact just yet - we haven't started formally using these yet. I think when we do integrate this into our NFRs we should only hard-code the start and endpoints and generate breakdowns for the steps in between (since the graph can potentially variate because of things like entity completeness and rpc queueing)

Reply:
But otherwise in terms of e2e, no impact

Reply:
If there are any changes to the names and/or semantics of the GDK events we should update this [documentation](https://docs.google.com/document/d/1RQYJ67ZXjHmv7mdi5Tx_sYxZHpbXynBQqSdE-eecies/edit#heading=h.nhkovyrruhdx) ideally.